benchmarks/operator_benchmark/pt/qpool_test.py

```diff
@@ -75,20 +97,29 @@ def forward(self):
 
 
 class QMaxPool2dBenchmark(_QPool2dBenchmarkBase):
-    def init(self, N, C, H, W, k, s, p, ceil, contig, dtype):
+    def init(self, N, C, H, W, k, s, p, contig, dtype):
         self.pool_op = torch.nn.MaxPool2d(kernel_size=k, stride=s, padding=p,
-                                          dilation=(1, 1), ceil_mode=ceil,
+                                          dilation=(1, 1), ceil_mode=False,
                                           return_indices=False)
         super(QMaxPool2dBenchmark, self).setup(N, C, H, W, dtype, contig)
 
 
 class QAvgPool2dBenchmark(_QPool2dBenchmarkBase):
-    def init(self, N, C, H, W, k, s, p, ceil, contig, dtype):
-        self.pool_op = torch.nn.AvgPool2d(kernel_size=k, stride=s, padding=p,
-                                          ceil_mode=ceil)
+    def init(self, N, C, H, W, k, s, p, contig, dtype):
+        self.pool_op = torch.nn.MaxPool2d(kernel_size=k, stride=s, padding=p,
```

Comment:
Should this be AvgPool2d? Seems to be a mismatch?

Reply:
You are right -- I noticed it yesterday, forgot to stack tthe changes :)